### PR TITLE
Feat: add console footer

### PR DESCRIPTION
**Pull request by TGlide**
## What does this PR do?

Adds a footer to the console

## Test Plan

Manual

## Related PRs and Issues

N/A

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by TorstenDittmann** [comment]
Bildschirm­foto 2023-02-06 um 09 17 51
it looks kinda weird on this screen, can you check with the design team?

**Comment by TGlide** [comment]
> Bildschirm­foto 2023-02-06 um 09 17 51 it looks kinda weird on this screen, can you check with the design team?

Fixed the spacing issue, but design requested aditional changes. GOing to convert this into draft because it may require changes in pink.

**Comment by ArmanNik** [comment]
image

We should add `u-cross-center` to align the icons and the text :+1:

**Review by ArmanNik — APPROVED** [review]

**Comment by TGlide** [comment]
> image
> 
> We should probably add `u-cross-center` to align the icons and the text 👍

Which browser is this happening on? I tested on Firefox and Chrome, and it's aligned 🤔

File: src/lib/stores/sdk.ts [hunk 1]
```diff
@@ -13,7 +13,8 @@ import {
     Users
 } from '@aw-labs/appwrite-console';
 
-const endpoint = VARS.APPWRITE_ENDPOINT ?? `${globalThis?.location?.origin}/v1`;
+export const endpoint = VARS.APPWRITE_ENDPOINT ?? `${globalThis?.location?.origin}/v1`;
```
Comment (TorstenDittmann) [inline src/lib/stores/sdk.ts:16]: I think it might make more sense to use `sdkForConsole.client.config.endpoint` for this instead of exporting 👍🏻

**Review by TorstenDittmann — APPROVED** [review]

File: src/lib/stores/sdk.ts (hunk 1 above)
Comment (TGlide) [reply src/lib/stores/sdk.ts]: Done!

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**